websites/T/Tellingtone/presence.ts

```diff
@@ -55,9 +55,12 @@ presence.on("UpdateData", async () => {
 			presenceData.buttons = [{ label: strings.buttonViewSeries, url: href }];
 	}
 	if (document.querySelector("div#Player")) {
-		presenceData.details = `${strings.currentlyListening} ${
-			document.querySelector<HTMLDivElement>("div.media-title").textContent
-		}`;
+		presenceData.details = strings.currentlyListening.replace(
+			"{0}{1}",
```

Comment:
that won't work, since `{0}` is optional, so it doesn't have to be there, nor does it have to be next to `{1}` in any way.

Reply:
Should be resolved in https://github.com/PreMiD/Presences/pull/6626/commits/e31734086b4dba81f471f70f6e15095945c28f5f ?